tslint-server/src/server.ts

```diff
@@ -71,6 +71,7 @@ namespace StatusNotification {
 	export const type = new server.NotificationType<StatusParams, void>('tslint/status');
 }
 
+let rootDir: string | null = null;
```

Comment:
Where is the variable `rootDir` set? In the changes I only see that `rootDir` is accessed